src/test/java/htsjdk/utils/SamtoolsTestUtils.java

```diff
@@ -0,0 +1,91 @@
+package htsjdk.utils;
+
+import htsjdk.samtools.util.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * Test utilities for running samtools from htsjdk tests.
+ */
+public class SamtoolsTestUtils {
+    private static final Log LOG = Log.getInstance(SamtoolsTestUtils.class);
+    private static final String SAMTOOLS_BIN_PROPERTY = "HTSJDK_SAMTOOLS_BIN";
+    public final static String expectedSamtoolsVersion = "1.9";
+
+    /**
+     * @return true if samtools is available
+     */
+    public static boolean isSamtoolsAvailable() {
+        final String binPath = getSamtoolsBin();
+        if (binPath == null) {
+            LOG.warn(String.format(
+                    "No samtools binary found. Set property %s to enable testing with samtools.",
+                    SAMTOOLS_BIN_PROPERTY));
+            return false;
+        }
+        final Path binFile = Paths.get(binPath);
+        if (!Files.exists(binFile)) {
+            throw new IllegalArgumentException(
+                    String.format(
+                            "%s property is set to non-existent file: %s", SAMTOOLS_BIN_PROPERTY, binFile));
```

Comment:
this error is confusing if you didn't set the property at all maybe?

Reply:
Fixed.